jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -450,6 +460,7 @@ pipeline {
                                     String bundleManifestUrl = buildManifestObj.getBundleManifestUrl(JOB_NAME, BUILD_NUMBER)
 
                                     triggerIntegrationTests(buildManifestUrl, 'linux', 'deb')
+                                    triggerSmokeTests(buildManifestUrl, 'linux', 'deb')
```

Comment:
Can we make these as parallel as well like above example?

Reply:
That makes sense. Working on it. Thanks!

Reply:
Updated.